prov/rxm/src/rxm.h

```diff
@@ -492,10 +502,15 @@ struct rxm_tx_rndv_buf {
 	/* Must stay at top */
 	struct rxm_buf hdr;
 
+	struct rxm_conn *conn;
 	void *app_context;
 	uint64_t flags;
 	struct fid_mr *mr[RXM_IOV_LIMIT];
+	size_t rndv_rma_index;
+	void *write_desc[RXM_IOV_LIMIT];
 	uint8_t count;
+	struct rxm_tx_base_buf *done_buf;
+	struct rxm_rndv_hdr remote_hdr;
```

Comment:
Separating this into two bufs would cause more branching and duplicate code to conditionally handle accessing/modifying the new struct.  Is that preferable to the additional size here?

Reply:
I'm more concerned about the readability than the size here.  So, duplicating code sounds worse.  Could you move the new fields into a substructure?  E.g.
```
struct {
    struct rxm_tx_base_buf *done_buf;
    struct rxm_rndv_hdr remote_hdr;
} write_rndv;
```